src/Raven.Client/ServerWide/Operations/CompareExchange.cs

```diff
@@ -2,7 +2,9 @@
 using System.Collections.Generic;
```

Comment:
I think that this should be moved elsewhere. Those operations are no longer ServerWide and should not be in Raven.Client.ServerWide namespace.

Reply:
Maybe in Raven.Client.Documents.Operations?